### DISCUSSION TITLE
Security Onion Setup Failing

### DISCUSSION THREAD

**Discussion by jrpayne30040** [Security-Onion-Solutions/securityonion#6630] [Unsupported Versions] [answered]
I am running the Security Onion setup up on Ubuntu 18.04. The setup willl not complete and says that Install had a problem and to check the log foe details. I have looked at the log but there is no specific error. Only one error that repeats 50 times that says the command failed with exit code 100. Like below

Reading package lists... (0)
Executing command with retry support: apt-get -y install docker-ce=5:20.10.5~3-0~ubuntu-bionic docker-ce-cli=5:20.10.5~3-0~ubuntu-bionic docker-ce-$
E: Packages were downgraded and -y was used without --allow-downgrades.
Results: Reading package lists...
Building dependency tree...
Reading state information...
python3-docker is already the newest version (2.5.1-1).
docker-ce-cli is already the newest version (5:20.10.5~3-0~ubuntu-bionic).
docker-ce-rootless-extras is already the newest version (5:20.10.5~3-0~ubuntu-bionic).
Suggested packages:
  aufs-tools cgroupfs-mount | cgroup-lite
The following packages will be DOWNGRADED:
  docker-ce
0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 13 not upgraded. (100)
Command failed with exit code 100; will retry in 10 seconds (1 / 50)...

Any ideas on how to find the command that's failing?

**Comment by weslambert** [comment, marked as answer]
Is this the first time you are attempting to install Security Onion on this machine, or have you run the installer previously?

**Reply by jrpayne30040** [reply]
This was the first time. I had tired. however, I am going to close the topic because, I just decided to go with something else.  Thank you for your reply!

**Reply by nutt318** [reply]
getting the same error you mention.

**Reply by dougburks** [reply]
@nutt318 From the [discussion guidelines](https://github.com/Security-Onion-Solutions/securityonion/discussions/1720):
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
